Change some sample disapprove comments to feel less 'stern'. (#1264)
exercism/website-copy: automated-comments/javascript/general/no_method.md

> In order to make the tests pass, you need to create a method called `%{method.name}`.

Reviewer: @kotp Any reason not to use the same phrasing a suggested earlier? So something like this:

```suggestion
In order to make the tests pass, there must be a method named `%{name}`.
```

> In order to make the tests pass, you need to create a method called `%{method.name}`.

Author: Same comment as above. I'd like the call to action to be clearer.
